recipes/app2.rb

```diff
@@ -124,3 +92,30 @@
   ]
   sensitive true
 end
+
+directory '/formsender'
+
+git '/var/lib/formsender' do
+  repository 'https://github.com/osuosl/formsender.git'
+  revision 'antoniagaete/RT_api'
+  notifies :build, 'docker_image[formsender]', :immediately
+end
+
+docker_image 'formsender' do
+  tag 'latest'
+  source '/var/lib/formsender'
+  action :nothing
+end
+
+formsender_env = data_bag_item('osl-app', 'formsender')
+
+docker_container 'support.osuosl.org' do
```

Comment:
Let's use this name instead.

```suggestion
docker_container 'formsender' do
```